src/input/Binding.zig

```diff
@@ -233,13 +233,14 @@ pub const Action = union(enum) {
     /// CSI header (`ESC [` or `\x1b[`).
     csi: []const u8,
 
-    /// Send an `ESC` sequence.
-    esc: []const u8,
+    /// Unless kitty disambiguate is active, send an `ESC` sequence.
+    esc_unless_kitty_disambiguate: []const u8,
```

Comment:
oh these names are also used by config -- should keep the old ones around then if we stick with this approach